samples/peripheral/radio_test/README.rst

```diff
@@ -61,7 +61,14 @@ nRF21540 front-end module
 
 .. include:: /includes/sample_dtm_radio_test_fem.txt
 
-The nRF21540 transmitted power gain, antenna and an activation delay can be configured using the user interface :ref:`radio_test_ui`.
+You can configure the nRF21540 front-end module (FEM) transmitted power gain, antenna and activation delay using the main shell commands of the :ref:`radio_test_ui`.
```

Comment:
```suggestion
You can configure the nRF21540 front-end module (FEM) transmitted power gain, antenna output and activation delay using the main shell commands of the :ref:`radio_test_ui`.
```
In the DTM doc, you had antenna *output*, so I guess it needs to be the same here.

Reply:
Added